File: Javascript/tasks/crossingSum.js
function crossingSum(matrix, a, b) {
    var s = 0;
    for (var i = 0; i < matrix[0].length; i++) {
        s += matrix[a][i];
    }
    for (var i = 0; i < matrix.length; i++) {
        s += matrix[i][b];
    }
    s -= matrix[a][b];
    return s;
}

File: tasks/crossingSum.py
def crossingSum(matrix, a, b):
  s = 0
  h, w = len(matrix), len(matrix[0])
  for i in range(0, w):
    s += matrix[a][i]
  for i in range(0, h):
    s += matrix[i][b]
  return s - matrix[a][b]
